src/Styles.jsx

```diff
@@ -1,20 +1,27 @@
-import styled from 'styled-components';
+import styled, { createGlobalStyle } from 'styled-components';
 
 const styles = {};
 
-styles.h4 = styled.h4`
+styles.global = createGlobalStyle`
```

Comment:
Please follow convention with capital component names. No need to use the styles object. What is it accomplishing?

convention is here: https://styled-components.com/

GlobalStyle
CenterDiv
StyledImage

But even then, they could be named better.

Price is not bad

SelectedDot

Does this make sense?

Anyone who used styled-components then reads this would be confused as to why we are breaking convention here.

Reply:
Yes, that makes a lot of sense. Keeps things organized in the components too. Thanks